supervisor/host/configuration.py

```diff
@@ -340,3 +360,7 @@ def _map_nm_vlan(inet: NetworkInterface) -> VlanConfig | None:
             return None
 
         return VlanConfig(inet.settings.vlan.id, inet.settings.vlan.parent)
+
+    @staticmethod
+    def _map_nm_multicast_dns(mode: int | None) -> MulticastDnsMode | None:
+        return _MULTICAST_DNS_VALUE_MODE_MAPPING[mode] if mode else None
```

Comment:
0 evaluates to false here, but is a valid mode with a mapping. I think we can simply use `get` here:

```suggestion
        return _MULTICAST_DNS_VALUE_MODE_MAPPING.get(mode)
```

Reply:
Good catch. Fixed